en/flight_modes/mission.md

```diff
@@ -162,3 +162,27 @@ Vehicles switch to the next waypoint as soon as they enter the acceptance radius
   - The equation is: 
     $$L_{1_{distance}}=\frac{1}{\pi}L_{1_{damping}}L_{1_{period}}\left \| \vec{v}_{ {xy}_{ground} } \right \|$$
 
+<span id="fixed_wing_mission_takeoff_landing"></span>
+## Fixed-wing Mission Takeoff/Landing
+
+Starting and ending flights with mission takeoff and landing is the recommended way of operating a plane autonomously. A detailed description of the executed logic during autonomous takeoffs and landings can be found
+in [Fixed-wing Takeoff](../config/flight_modes/takeoff.md#fixed_wing) and [Fixed-wing Takeoff](../config/flight_modes/land.md#fixed_wing).

```

Comment:
Need to fix the link once we get the landing part merged https://github.com/PX4/PX4-user_guide/pull/2245)